openhands/llm/llm.py

```diff
@@ -235,9 +235,16 @@ def wrapper(*args, **kwargs):
             # NOTE: this setting is global; unlike drop_params, it cannot be overridden in the litellm completion partial
             litellm.modify_params = self.config.modify_params
 
+            # if we're not using litellm proxy, remove the extra_body
+            if 'litellm_proxy' not in self.config.model:
+                kwargs.pop('extra_body', None)
+
             # Record start time for latency measurement
             start_time = time.time()
             # we don't support streaming here, thus we get a ModelResponse
+            logger.info(
```

Comment:
if this is debug let's make it debug
```suggestion
            logger.debug(
```

Reply:
ahh yes!